dashboard/config/linux/bits/android-5.10.yml

```diff
@@ -4,10 +4,10 @@
 # See https://source.android.com/setup/build/building-kernels
 # for details on how to checkout and build Android kernel.
 
-# Following branch: android12-5.10-lts
+# Following branch: android13-5.10-lts
```

Comment:
How important is it to switch from android12 to android13? Do they have different kernels or configs?

Changing branch on the fly may affect some of the bisection and tree origin detection logic. But I am not sure right now how exactly.

Reply:
If the kernel is different, then we also need to switch internal syz-ci configs to android13. Otherwise they will still test android12.

Reply:
android13 has more features and a higher EOL than android12, so yes, the kernels and configs may be different but still based on 5.10. See  https://source.android.com/docs/core/architecture/kernel/android-common#kmi-branch-lifecycle . I think it's better to test on the latest android version, but let's see what @lag-linaro thinks too.

Reply:
It's okay to make the switch, but as Dmitry says, you need to update the configs as well.

Why don't you handle that as part of an othognal pull-request?

Let's only create `v6.1` in this one.

Reply:
> Why don't you handle that as part of an othognal pull-request?

syz-ci configs are stored elsewhere (internally)

Reply:
Presumably the kernel configs will also need to be rebuilt too.

Reply:
Indeed, it seems there's a difference between the dashboard's kernel configs and the  syz-ci kernel configs. Take https://syzkaller.appspot.com/bug?id=0f1e1644d7ed28d1d925d6c9e2e2c0c1c18cb794 for example, it uses a v5.10.177 kernel, while the kernel version specified by android-5.10.yml is 8d21bcc704ea1 (v5.10.82). Wondering why we define a kernel config for dashboards now.